frontend/src/sass/elements/_buttons.scss

```diff
@@ -54,6 +54,9 @@
     &:active {
       background-color: $fs-green-darker;
     }
+    &:disabled {
```

Comment:
did disabled end up getting used? I thought it wasn't accessible?

Reply:
added task in #285 to remove this